src/components/Main/Stats/CaptionedGraphic.tsx

```diff
@@ -8,10 +9,20 @@ interface Props {
 
 function CaptionedGraphic(props: Props) {
   return (
-    <div>
-      <img src={props.img} alt={props.alt} />
-      <p>{props.caption}</p>
+    <div className="col-md-4 col-sm-6">
+      <div className="d-flex justify-content-center">
+        <img src={props.img} width="140px" height="140px" alt={props.alt} style={{objectFit: "contain"}}/>
+      </div>
+      <p className="text-dark-grey text-center">{props.caption}</p>
     </div>
+    // <div className="col-sm-4 col-6">
```

Comment:
remove anything that is commented out -- if u don't need it, then dont worry abt it